.changeset/six-poems-press.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-node': patch
+---
+
+[emailMatchingUserEntityProfileEmail resolver]: Try also matching emails with plus addressing removed.
```

Comment:
```suggestion
The `emailMatchingUserEntityProfileEmail` sign-in resolver will now also try matching emails with plus addressing removed.
```

Not really a format we use much 😅

Reply:
Haha, should've looked at some other ones, sorry 🙈
Have updated the PR